python/nammu/view/NewAtfView.py

```diff
@@ -176,7 +176,7 @@ def create_project_list():
             Prepares list of projects and subprojects ordered with the default
             one first.
             '''
-            default_project = self.projects['default'].split('/')[0]
+            default_project = self.projects['default'][0].split('/')[0]
```

Comment:
This line is now correct, but we'll need to think about users who already have a config file that is wrong with a preferred project as a string instead of a list. This makes things a bit awkward. That change should go on the utils file, when the config file from the `resources` folder is compared with the one the user already has. We need to check for the default project and make it a list if it's a string.

Reply:
Resolved with the creation of a new method in `utils` in commit 77445464 which tests on file load for a default projects string and if present adds it to a list.